ci/cpu/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash -ex
+
+export PATH=/conda/bin:/usr/local/cuda/bin:$PATH:$WORKSPACE
+export HOME=$WORKSPACE
+
+BRANCH="$(git rev-parse --abbrev-ref HEAD)"
+if [[ "$BRANCH" != "main" ]]; then
+  echo 'Branch release not main, exiting...';
+  exit 1;
```

Comment:
This should probably exit with `0`, right? It's fine if it gets run on other branches, it just won't publish anything.